scss/components/molecules/_navigation.scss

```diff
@@ -0,0 +1,380 @@
+@charset 'UTF-8';
+
+///
+/// NAVIGATION COMPONENT
```

Comment:
This file is very long and contains three different navigational items. Main / Side / Mega. It would be best to split these up in to their own files.